packages/shared-state-pirania/files/etc/shared-state/publishers/shared-state-publish_vouchers

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/lua
+
+--! LibreMesh
+--! Copyright (C) 2019  Marcos Gutierrez <gmarcos@altermundi.net>
+--! Copyright (C) 2019  Luandro <luandro@gmail.com>
+--!
+--! This program is free software: you can redistribute it and/or modify
+--! it under the terms of the GNU Affero General Public License as
+--! published by the Free Software Foundation, either version 3 of the
+--! License, or (at your option) any later version.
+--!
+--! This program is distributed in the hope that it will be useful,
+--! but WITHOUT ANY WARRANTY; without even the implied warranty of
+--! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--! GNU Affero General Public License for more details.
+--!
+--! You should have received a copy of the GNU Affero General Public License
+--! along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+local fs = require("nixio.fs")
+local JSON = require("luci.jsonc")
+
+local function shell(command)
+    local handle = io.popen(command)
+    local result = handle:read("*a")
+    handle:close()
+    return result
+end
+
+local vouchers = JSON.parse(shell('voucher list_vouchers')) or {}
+local result = {}
+
+for _, voucher in pairs(vouchers) do
+    result[voucher.voucher] = voucher
+end
+
+io.popen("shared-state insert pirania", "w"):write(JSON.stringify(result))
```

Comment:
Add a new line at the end of the file please

Reply:
done in 9c23f93ecfabab041139f534d3d731e88b9398b9